src/twisted/conch/test/test_scripts.py

```diff
@@ -4,56 +4,52 @@
 """
 Tests for the command-line interfaces to conch.
 """
+from unittest import skipIf
 from twisted.python.reflect import requireModule
+from twisted.python.test.test_shellcomp import ZshScriptTestMixin
+from twisted.scripts.test.test_scripts import ScriptTestsMixin
+from twisted.trial.unittest import TestCase
 
-if requireModule('pyasn1'):
-    pyasn1Skip = None
-else:
-    pyasn1Skip =  "Cannot run without PyASN1"
+doSkip = False
+skipReason = ""
 
-if requireModule('cryptography'):
-    cryptoSkip = None
-else:
+if not requireModule('pyasn1'):
+    doSkip = True
+    skipReason = "Cannot run without PyASN1"
+
+if not requireModule('cryptography'):
+    doSkip = True
     cryptoSkip = "can't run w/o cryptography"
 
-if requireModule('tty'):
-    ttySkip = None
-else:
+if not requireModule('tty'):
+    doSkip = True
     ttySkip = "can't run w/o tty"
 
 try:
-    import Tkinter
+    import tkinter
```

Comment:
This seems like it either breaks or fixes the tests on case-sensitive filesystems (probably fixes them). It might be worth opening another trac ticket for this (both tickets can be listed on the merge commit's `fixes:` line) and also fixing the other two capital-T `Tkinter` imports in `examples/tkinterdemo.py` and `howto/choosing-reactor.rst`.

Reply:
No, this is a Python 2 vs. Python 3 thing.  On Python 2, the module was named `Tkinter`, but on Python 3, it was changed to `tkinter`.  

I can open a separate PR for the example and howto.